pages/cql/OpioidCDS_DSTU2_REC_10.cql

```diff
@@ -55,15 +55,14 @@ define "Validate Trigger Event":
   )
 
 define "Inclusion Criteria":
-  AgeInYears() >= 18
+  AgeInYearsAt(Now()) >= 18
     and "Validate Trigger Event"
```

Comment:
What prompted the change here? This is using a DateTime (Now()), and so will result in the patient's birth _time_ being considered if it is present. That isn't the intent, so the AgeInYears() is preferred.